setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+
+setup(
+    name="FILL",
+    version="FILL",
+    license="FILL",
+    author="FILL",
+    author_email="FILL",
+    packages=["ZeroSync"],
+    package_dir={"ZeroSync": "src/ZeroSync"},
```

Comment:
Would it suffice to change this line to get rid of the `ZeroSync` folder in `src`?

Reply:
I don't know it might introduce some problems but you can try